src/js/jsx/sections/libraries/LibrariesPanel.jsx

```diff
@@ -106,47 +117,49 @@ define(function (require, exports, module) {
          * 
          * @return {ReactComponent}
          */
-        _containerContents: function () {
+        _renderLibrariesContent: function () {
             var libraryStore = this.getFlux().store("library"),
                 connected = libraryStore.getConnectionStatus(),
                 containerContents;
 
             if (connected) {
-                var libraries = this.state.libraries,
-                    currentLibrary = libraries.get(this.state.selectedLibrary);
-
-                containerContents = this.props.visible && !this.props.disabled && (
-                    <div>
-                        <div className="formline">
-                            <LibraryList
+                if (this.props.visible && !this.props.disabled) {
```

Comment:
I gave it a look and turns out you have to keep both, because the panel might be visible while it is disabled for unsupported document.